client/app/components/sections/Topbar/Topbar.css

```diff
@@ -54,16 +54,29 @@
     }
   }
 
+  & button {
```

Comment:
Yes, makes sense. Just for clarity, would it be possible to make a `resetStylesLeakingFromRails.css` (naming left as exercise) and include it to root components?